docs/resources/Channel.md

```diff
@@ -975,7 +975,7 @@ Returns all pinned messages in the channel as an array of [message](#DOCS_RESOUR
 Pin a message in a text channel. Requires the `MANAGE_MESSAGES` permission. Returns a `204 No Content` on success.
 
 > info
-> The maximum number of pinned messages per text channel is currently set to 50.
+> The maximum number of pinned messages per text channel and news channel is currently set to 50.
```

Comment:
maybe just leave it as "per channel" tbh? saying the isn't really necessary, as there are no different limits for other channel types

Reply:
I feel it would be worth stressing that it is for text channels and news channels specifically, just since the documentation is not overly clear on channel-specific features. At the worst, it can prevent someone asking redundant questions in issues because of a lack of explicit clarity.

Reply:
I feel it would be worth stressing that it is for text channels and news channels specifically, just since the documentation is not overly clear on channel-specific features. At the worst, it can prevent someone asking redundant questions in issues because of a lack of explicit clarity.